index.html

```diff
@@ -419,5 +427,13 @@ <h2 class="form-header">Contact me</h2>
         </div>
       </section>
     </div>
+    <div class="overlay-menu hidden">
+      <a><bold>&times;</bold></a>
+      <a href="#portfolio" style="display: block; color: white;"><b>Portfolio</b></a>
+            <a href="#about" style="display: block; color: white;"><b>About</b></a>
+            <a href="#contact" style="display: block; color: white;"><b>Contact</b></a>
+    </div>
+
+    <script src="./main.js"></script>
```

Comment:
- [ ] Good job making most of the changes highlighted by the previous reviewer 👏 . However, there's one left, which happens to be the close button.
- I could cancel the menu pop up by clicking the empty space by the top right corner, but its not user diecting and does not follow requiements. Kindly add button, which might be a case of `CSS`, to give a better user experince 👍 

## App view:
![image](https://user-images.githubusercontent.com/50990925/231246141-a0daf484-77db-43f6-9210-1d4fe6cbabb3.png)

## Figma:
![image](https://user-images.githubusercontent.com/50990925/231246739-9e5632dd-9c0f-4c34-8984-2888c0ca9c41.png)